src/SFA.DAS.QnA.Application/Validators/MaxWordCountValidator.cs

```diff
@@ -15,7 +15,9 @@ public class MaxWordCountValidator : IValidator
 
             if (!string.IsNullOrEmpty(text))
             {
-                var wordCount = text.Split(new[] { " " }, StringSplitOptions.RemoveEmptyEntries).Length;
+                char[] splitters = new char[] { ' ', '\r', '\n' };
+
+                var wordCount = text.Split(splitters, StringSplitOptions.RemoveEmptyEntries).Length;
```

Comment:
Please add tests for any code changes whenever possible.

Reply:
There are tests for this validator and they are passing.

Reply:
Those tests are for the existing code. The tests for your newly added logic won't exist unless you write them yourself.